Update Docker database initialisation (#2192)
wmde/fundraising-application: doc/HOWTO_Import_Geodata.md

# How to Import the Third Party Geodata

The Geodata zip contains 2 sql files, a small one with community names, and a large one which includes streets.

1. Copy the smaller sql file into /build/database.

> 2. Rename it to `00.GEodata.sql`. This is needed because Docker runs the files in order, and the Geodata import needs to run before the Schema.

Reviewer: ```suggestion
2. Rename it to `00.Geodata.sql`. This is needed because Docker runs the files in order, and the Geodata import needs to run before the Schema.
```

3. Remove the database volume with the command:
   
   ```$ docker volume rm APP_DIRECTORY_db-storage```
   
   APP_DIRECTORY is the directory name where you have checked out the `fundraising-application` code. You can check for the volume name by listing all docker volumes with the command `$ docker volume ls`.

> 4. Run `$ docker-compose up`. When importing the Geotata this might take a little while as it imports.

Reviewer: ```suggestion
4. Run `$ docker-compose up`. When importing the Geodata, this might take several minutes.
```

# How to Import the Third Party Geodata

The Geodata zip contains 2 sql files, a small one with community names, and a large one which includes streets.

1. Copy the smaller sql file into /build/database.

> 2. Rename it to `00.[FILENAME].sql`. This is needed because Docker runs the files in order, and the Geodata import needs to run before the Schema.

Reviewer: ```suggestion
2. Rename it to `00.Geodata.sql`. This is needed because Docker runs the files in order, and the Geodata import needs to run before the Schema.
```
I'm all for indicating that a file name is irrelevant, but using a placeholder here might trip up inexperienced developers who might name that file literally `00.[FILENAME].sql`

> 3. Remove the database volume `$ docker-compose volume rm fundraising-app_db-storage`.

Reviewer: ```suggestion
3. Remove the database volume with the command  
    `docker volume rm APP_DIRECTORY_db-storage`. 
    APP_DIRECTORY is the directory name where you have checked out the `fundraising-application` code. You can check for the volume name by listing all docker volumes with the command  
    `docker volume ls`
```
